Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.java

```diff
@@ -57,6 +57,8 @@ public ScheduleAdapterItem(ScheduleDatabase.ScheduleRow scheduleRow,
             roomOrder = 2;
         } else if ("CYCLORAMA".equals(scheduleRow.room)) {
```

Comment:
We can not compare with room names anymore. Sorting Logic : 
Tracks will have a priority 1,2 and 3. we can use the same priority to sort the talks for the same time-slot. Stage Names Now:  Stage 1 (Virginia Wimberly Theatre), then Stage 2 (Nancy and Edward Roberts Studio Theatre) and then Workshops (Carol Deane).

Reply:
I can also add a sort order field if that helps.

Reply:
Denormalized a "trackSortOrder" field into the event/session.
The sortOrder field was already in the track object.

Reply:
using this field now